Livl-Corporation/livl-madness
# [FEAT / FIX] Add livl product and other behaviour

### What has changed ?

- Add new custom livl products
- Fixed their spawn position height
- Translate their name to make the game easier to play
- Facilitate DX with a script for livl product prefabs

### How to test ?
- Just start the game.
- You must be able to scan any items
- All the items should be named in french in your livl app 🙌

close #38 
close #39
